control/grpc.py

```diff
@@ -323,7 +326,10 @@ def create_bdev(self, anagrp: int, name, uuid, rbd_pool_name, rbd_image_name, bl
                 uuid=uuid,
             )
             self.bdev_cluster[name] = cluster_name
-            self.logger.info(f"bdev_rbd_create: {bdev_name}")
+            #TODO - delete this like bdev_cluster
```

Comment:
Sure 😊 , could you add removal of the `bdev_params`  key [here](https://github.com/ceph/ceph-nvmeof/blob/022e30008a31a83eaed645a9699ee90b7ed3cf28/control/grpc.py#L392)?